build/ps-entry.sh

```diff
@@ -328,6 +325,11 @@ if [ "$originalArgOne" = 'mongod' ]; then
 	if [ -f ${MONGO_SSL_DIR}/ca.crt ]; then
 		CA=${MONGO_SSL_DIR}/ca.crt
 	fi
+
+	sslMode="$(_mongod_hack_have_arg '--sslPEMKeyFile' "$@" && echo 'preferSSL' || echo 'disabled')" # "BadValue: need sslPEMKeyFile when SSL is enabled" vs "BadValue: need to enable SSL via the sslMode flag when using SSL configuration parameters"
+	_mongod_hack_ensure_arg_val --sslMode "$sslMode" "${mongodHackedArgs[@]}"
```

Comment:
why `mongodHackedArgs` needed?
```suggestion
	_mongod_hack_ensure_arg_val --sslMode "$sslMode" "$@"
```